roles/keystone-setup/tasks/main.yml

```diff
@@ -27,7 +52,25 @@
                     internal_url={{ item.internal_url }}
                     admin_url={{ item.admin_url }}
                     region=RegionOne
-                    token={{ secrets.admin_token }}
+                    token={{ admin_token }}
   # TODO refactor keystone.services data to be easier to pull out individual services
   with_items: keystone.services
   when: endpoints[item.name] is defined and item.name == 'keystone'
+
+- name: remote admin_token from keystone api
+  lineinfile: dest=/etc/keystone/keystone.conf
+              line="admin_token = {{ admin_token }}"
```

Comment:
Actually running the template would be difficult due to variables not in this role. Given that we add the line right after the [DEFAULT] header, and we're removing the fully listed line, I think it's fine to use lineinfile to remove it. Any other occurrence of admin_token will have come from the template delivered in the keystone role.